docs/compliance/index.md

```
@@ -201,8 +201,7 @@ The report includes version, platform, test summary, and digests.
 
 ---
 
-## 12. Future Work
-
+12. Future Work
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 12; Style: 1/1/1

(MD029, ol-prefix)